lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -105,9 +105,10 @@ func (handler *ContainersHandlersImpl) CreateHandler(params containers.CreatePar
 				},
 			},
 		},
-		Key:      pem.EncodeToMemory(&privateKeyBlock),
-		LayerID:  *params.CreateConfig.Image,
-		RepoName: *params.CreateConfig.RepoName,
+		Key:        pem.EncodeToMemory(&privateKeyBlock),
+		LayerID:    *params.CreateConfig.Image,
+		RepoName:   *params.CreateConfig.RepoName,
+		StopSignal: *params.CreateConfig.StopSignal,
```

Comment:
I think we should associate this per Session rather than per Executor. If we can have multiple Sessions per Executor then we should allow those sessions to have distinct stop signals.